android/engine/src/test/java/org/smartregister/fhircore/engine/sync/SyncBroadcasterTest.kt

```diff
@@ -136,16 +136,20 @@ class SyncBroadcasterTest : RobolectricTest() {
 
     Assert.assertEquals(resourceTypes, syncParam.keys.toTypedArray().sorted())
 
-    syncParam.keys.filter { it.isIn(ResourceType.Binary, ResourceType.StructureMap) }.forEach {
-      Assert.assertTrue(syncParam[it]!!.containsKey("_count"))
-    }
+    syncParam
+      .keys
+      .asSequence()
+      .filter { it.isIn(ResourceType.Binary, ResourceType.StructureMap) }
+      .forEach { Assert.assertTrue(syncParam[it]!!.containsKey("_count")) }
 
-    syncParam.keys.filter { it.isIn(ResourceType.Patient) }.forEach {
+    syncParam.keys.asSequence().filter { it.isIn(ResourceType.Patient) }.forEach {
```

Comment:
Using as sequence here is not useful. You need to check the kind of operations that are used after the sequence. In this case we filter then run for each statement so no impact at all.